docs/src/main/asciidoc/command-mode-reference.adoc

```diff
@@ -137,6 +137,22 @@ thread (A non-command mode application is essentially just running an applicatio
 If you want to shut down a running application and you are not in the main thread, then you should call `Quarkus.asyncExit`
 in order to unblock the main thread and initiate the shutdown process.
 
+=== Running the application 
+
+To run the command mode application on the JVM, first build it using `mvnw package` or equivalent. 
+
+Then launch it:
+
+```
+java -jar target/quarkus-app/quarkus-run.jar
+```
```

Comment:
Please use `[source,shell]` AsciiDoc.

Reply:
Good spot, thanks. I've done that on the two code blocks, and also switched from markdown to asciidoc syntax.